frontend/src/tests/application-tests.js

```diff
@@ -6,20 +6,72 @@ import {
     postingPropTypes,
 } from "./utils";
 import { databaseSeeder } from "./setup";
+import axios from "axios";
+import fs from "fs";
+import path from "path";
+import md5 from "md5";
 
 export function applicationsTests({ apiGET, apiPOST }) {
     let session;
     let position;
     let posting = {};
     let adminUser;
-    let surveyData;
+
+    const BACKEND_BASE_URL = "http://backend:3000";
```

Comment:
(nb) This can be potentially be imported from `utils` file if we export the `URL` const.